dashboard/src/components/DeploymentFormBody/AdvancedDeploymentForm.v2.tsx

```diff
@@ -7,7 +7,7 @@ import "ace-builds/src-noconflict/theme-xcode";
 export interface IAdvancedDeploymentForm {
   appValues?: string;
   handleValuesChange: (value: string) => void;
-  children: JSX.Element;
+  children?: JSX.Element;
```

Comment:
Why would we ever have a form without any children? Seems like we're throwing away some benefit of the type checker, but maybe I'm missing something?